### DISCUSSION TITLE
ubuntu docker run

### DISCUSSION THREAD

**Discussion by c2006878** [nocodb/nocodb#7217] [Q&A]
I am using Docker on Windows and everything is running normally, but I am using the command (docker run -d --name nocodb-mysql-v2 --add-host=host.docker.internal:host-gateway -v "$(pwd)"/nocodb:/usr/app/data/ -p 8080:8080 -e NC_DB="mysql2://host.docker.internal:3306?u=root&p=123456&d=noco" -e NC_AUTH_JWT_SECRET="569a1821-0a93-45e8-87ab-eb857f20a010" nocodb-local) in Ubuntu to create projects and tables, but cannot delete them, and then it prompts me for a network error. And even after personally deleting the corresponding table from the database, I can still see it on the webpage. Why is this, how should I solve it?

**Comment by mertmit** [comment]
What happens when you try to remove table, can you share error messages (UI or docker logs) if any?
